web/template/layout/mlpa-header.gohtml

```diff
@@ -1,23 +1,23 @@
 {{ define "mlpa-header" }}
     <span class="govuk-caption-l">
-      Date of birth {{ (date .Lpa.Application.DonorDob "2 January 2006") }} | Postcode
-      {{ .Lpa.Application.DonorAddress.Postcode }}
+      Date of birth {{ (date .CaseSummary.Lpa.Application.DonorDob "2 January 2006") }} | Postcode
+      {{ .CaseSummary.Lpa.Application.DonorAddress.Postcode }}
     </span>
-    <h1 class="govuk-heading-xl">{{ .Lpa.Application.DonorFirstNames }} {{ .Lpa.Application.DonorLastName }}</h1>
+    <h1 class="govuk-heading-xl">{{ .CaseSummary.Lpa.Application.DonorFirstNames }} {{ .CaseSummary.Lpa.Application.DonorLastName }}</h1>
 
     <nav class="moj-sub-navigation govuk-!-margin-bottom-0" aria-label="linked cases">
         <ul class="moj-sub-navigation__list">
             {{ range .SortedLinkedCases }}
                 <li class="moj-sub-navigation__item">
-                    <a class="moj-sub-navigation__link" {{ if eq $.Lpa.UID .UID }}aria-current="page"{{ end }} href="{{ prefix (printf "/lpa/%s" .UID ) }}">
-                        {{ template "status-tag" (ToLower $.Lpa.Status) }} {{ ToUpper .Subtype }} {{ .UID }}
+                    <a class="moj-sub-navigation__link" {{ if eq $.CaseSummary.Lpa.UID .UID }}aria-current="page"{{ end }} href="{{ prefix (printf "/lpa/%s" .UID ) }}">
+                        {{ template "status-tag" (ToLower $.CaseSummary.Lpa.Status) }} {{ ToUpper .Subtype }} {{ .UID }}
                     </a>
                 </li>
             {{ end }}
         </ul>
     </nav>
 
-    {{ template "mlpa-case-summary" }}
+    {{ template "mlpa-case-summary" $ }}
```

Comment:
is this $ just a typo?

Reply:
No, it passes all the variables in the current scope to the child template "mlpa-case-summary". I just added it so this was explicit.

Reply:
I took it out.

Reply:
I realised I needed to add this back in, as it doesn't work without it :)